src/app/shared/interceptors/x-app.interceptor.ts

```diff
@@ -6,22 +6,20 @@ import {
 } from '@angular/common/http';
 import { Inject, Injectable } from '@angular/core';
 import { Observable } from 'rxjs';
-import { ANALYTICS_RECOMMENDER_URL } from '../../space/app-launcher/shared/analytics-url.service';
 import { FABRIC8_FORGE_API_URL } from '../runtime-console/fabric8-ui-forge-api';
 
 @Injectable()
 export class XAppInterceptor implements HttpInterceptor {
 
   constructor(
-    @Inject(FABRIC8_FORGE_API_URL) private forgeApiUrl: string,
-    @Inject(ANALYTICS_RECOMMENDER_URL) private recommenderApiUrl: string
+    @Inject(FABRIC8_FORGE_API_URL) private forgeApiUrl: string
   ) {}
 
   intercept(request: HttpRequest<any>, next: HttpHandler): Observable<HttpEvent<any>> {
     const url = request.url;
 
     // Attach a X-App to all requests that don't have one
-    if ((url.startsWith(this.forgeApiUrl) || url.startsWith(this.recommenderApiUrl)) && !request.headers.has('X-App')) {
+    if ((url.startsWith(this.forgeApiUrl)) && !request.headers.has('X-App')) {
```

Comment:
Extra pair of parenthesis here. It can be just - `if (url.startsWith(this.forgeApiUrl) && !request.headers.has('X-App'))`